core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/parser/YamlSignerParser.java

```diff
@@ -26,13 +26,16 @@
 
 import com.fasterxml.jackson.core.JsonParseException;
 import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
 import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
 
 public class YamlSignerParser implements SignerParser {
 
   public static final YAMLMapper YAML_MAPPER =
       YAMLMapper.builder()
           .addModule(new SigningMetadataModule())
+          .disable(YAMLGenerator.Feature.WRITE_DOC_START_MARKER)
+          .disable(YAMLGenerator.Feature.USE_NATIVE_TYPE_ID)
```

Comment:
Why it this needed?

Reply:
Native type id would generate `!<file-keystore>`. Disabling it will generate `type="file-keystore"`